security/pkg/pki/ca/ca.go

```diff
@@ -194,6 +209,7 @@ func NewSelfSignedIstioCAOptions(ctx context.Context,
 				return fmt.Errorf("failed to create CA KeyCertBundle (%v)", err)
 			}
 			// Write the key/cert back to secret, so they will be persistent when CA restarts.
+			// This uses the old-style Istio names, for compatibility.
```

Comment:
And the root-cert instead of CA.crt, etc - the non-standard names for the keys. 

I would like to keep the old in istio-ca-secret - it's been the original secret from Istio 0.2 ( or around that ), and we gain nothing by messing with that, it is the simplest of all and used by 99% of users.

For ca-certs - we can be more aggressive and move to the standard naming ( TLS.crt, etc) - update docs and remove/change to the new style, even migrate by copying over from old names to new names (of the keys/files)

At this point - I wish we picked a new, separate secret to avoid the confusion. It looked easier to keep cacerts, but not sure anymore.